app/views/keywords/_upload_form.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for(:csv_upload_form,
+             as: CsvUploadForm,
+             url: keywords_path,
+             method: :post,
+             class: 'form-csv-upload py-4 d-flex justify-content-center'
```

Comment:
Just FYI, in IC we can use the inline CSS class like this, but in our client project, it depends on each project codebase, some projects will not allow to have `inline CSS`, all styles will be written in the CSS components

Reply:
thanks for the heads up 👍